### ISSUE TITLE
v1.3.3 Changing the Anchor on pictures TWICE duplicates the picture entry in the PicList

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#296]
v1.3.2 Changing the Anchor on pictures duplicates the picture entry in the PicList

The original anchor for this picture was 5:5. I then changed the anchor to 5:9, and then 5:13, and then finally to 5:2.
This caused PTXprint piclist editor to duplicate the entry 3 times! 

Here is what I'm seeing in the file itself:
![image](https://user-images.githubusercontent.com/65363206/99120036-305cb100-25c8-11eb-88f9-0258299cad63.png)

I suspect the issue is with PTXprint using the Anchor value as the means of organizing the picture list table. This just isn't a safe way to work it, apparently. We need the pictures numbered independently of any of the values in their settings. So that if I change the Anchor value of picture number 8, you KNOW that the new anchor value is NOT an addition, but simply a change to #8's value. It looks like this is most often a problem when I change the anchor value BACKWARDS to an earlier verse number. This seems to confuse the table, because now there was a "new" picture to go in a 5:2, there just happened to also be a 5:13 entry, etc. 

I suspect this is what others have been encountering too.

**Comment by davidc86** [comment]
v1.3.3
Just got this changing an anchor TWICE. (i.e. changed it once, ran PRINT, wasn't right, so I changed  it again, ran PRINT) First time it changed just fine, it was on the second change that I got this: 

![image](https://user-images.githubusercontent.com/65363206/99305182-69dc2900-2821-11eb-8fc3-3e4ab916bcc9.png)

Even though the Table still shows it as only ONE illustration entry: 

![image](https://user-images.githubusercontent.com/65363206/99305244-811b1680-2821-11eb-8738-dbf5e6052b90.png)

But if I close PTXprint, I see these entries in the PicList file: 

![image](https://user-images.githubusercontent.com/65363206/99305399-b889c300-2821-11eb-8d55-e616d4194c45.png)

So, clearly PTXprint's piclist file says there are TWO entries now, but the PTXprint Table only showed ONE.
 
(NOTE: If I reopen PTXprint, it will now see 2 illustrations there, because it is reading in this error from the PicList file. So, I have to delete the extra entry first before reopening PTXprint.) 

So, the pattern I'm seeing is **_if we make multiple changes to the anchor for a single illustration_** then PTXprint gets confused. 

Maybe the ENTIRE TABLE needs to be written out and read back in whenever changes is made to an illustration. Maybe this would insure that PTXprint is not getting confused with whether this is a new picture or simply a change to an existing one. (There is probably a better way :) just trying to think through what is happening here.)
